pkg/agent/openflow/pipeline_windows.go

```diff
@@ -42,8 +42,238 @@ const (
 
 	// snatCTMark indicates SNAT is performed for packets of the connection.
 	snatCTMark = 0x40
+
+	// trafficFromGatewayAndSNATMark indicates SNAT is performed on traffic from antrea-gw0.
+	trafficFromGatewayAndSNATMark = 0x20001
 )
 
+//var (
+//	// snatMarkRange takes the 17th bit of register marksReg to indicate if
+//	// the packet needs to be SNATed with Node's IP or not.
+//	snatMarkRange = binding.Range{17, 17}
```

Comment:
How it is different from snatPktMarkRange? I feel you need to clearly explain why add a new mark.